### ISSUE TITLE
make sure our TWYW distribution is what we want

### ISSUE THREAD

**Issue by nobodxbodon** [gratipay/inside.gratipay.com#986]
Reticket from #984:

@mattbk 
> I would like to see money put toward TWYW...but I think there is some social pressure holding people back from taking more than @whit537 does. I know that TWYW is take what you want, but do you see what I mean?

> Not implying that Chad wants to hold people back--I think he wants the opposite.

@nobodxbodon:
>@mattbk great point you raised. IMO it's one of the most critical issues in current twyw model.

> One thought, we use group intelligence instead of personnal intelligence to help this issue:

>    1. a list of contributors is assembled for the pool we want to distribute
>    2. every contributor in the list who want to vote, put a number/percentage to any contributors on that list (need to sum to 100%)
>    3. get average of the votes above, which will be used to calculate the final number
>    4. of course not everyone will take the exact number in the final result, but that will ease the social pressure at least IMO
>   5. @whit537 is the last one who takes, to make it even less pressure for all others to take :)

@mattbk 
> Actually, I think he's the last anyway if TWYW is still operating under "kids eat first," where the newest members of the team project get first crack at taking.

**Comment by whit537** [comment]
> I think there is some social pressure holding people back from taking more than @whit537 does.

I may actually drop to 1¢ to indicate that I don't need the money right now, which is true. I've got a consulting contract that looks pretty steady. I work three hours every morning on that six days a week, and that is plenty for my family right now.

**Comment by whit537** [comment]
> I may actually drop to 1¢

[Done](https://gratipay.com/Gratipay/distributing/)! :-)

![screen shot 2017-01-16 at 3 58 25 pm](https://cloud.githubusercontent.com/assets/134455/21998735/a22cf52c-dc04-11e6-9f4e-81a36ccddfc5.png)

**Comment by whit537** [comment]
If some of that $50/wk would actually mean a lot to someone in the @gratipay crew then by all means please up your take. Let's get some momentum going here! 💸 💸 💸 💸 💃

**Comment by whit537** [comment]
> I work three hours every morning on that six days a week

P.S. It's a habit I picked up in #867. After blowing off paid work during #757, I knew I couldn't do the same two months in a row. Due to jet lag I was waking up early anyway, so I just got in the habit of putting three hours on the clock first thing. By 9:00 AM I had my chores out of the way and could focus on Gratipay the whole rest of the day. I did that the whole two weeks I was in SF, and it worked so well that I kept up the habit upon returning home. I don't even use an alarm, I just go to bed by 10:00 and then I pop awake at 4:00 or 5:00 and jump right on the clock. Done by 8:00 at the latest, breakfast and shower and even if I catch a bus I'm on Gratipay by 10:00.

tl;dr I'm ready to rock 2017 pretty much full-time on Gratipay, even though all of #637 #836 #388 fell through. 💃 🎸

**Comment by nobodxbodon** [comment]
Nice moves @whit537 and thanks for sharing your schedule and plan! Still IMO we'll need some adjustments in the twyw model, mainly to relieve the social pressures as much as possible. One possibility I thought about earlier is TWYW based on collaborative review of the distribution, something like "Take What All Want You to Take".

**Comment by whit537** [comment]
Yeah, that's a different model. I described it in "[3 lessons from Gratipay's take-what-you-want compensation experiment](https://opensource.com/open-organization/16/7/compensating-employees-letting-them-take-what-they-want)" as rankings-based. We tried that briefly before switching to twyw. The social pressure is an essential part of twyw. :-)

What do you want to take, @nobodxbodon? Personally I see you and @mattbk doing the most work besides me—probably even as much as me. I see another tier of folks doing good work but a bit less of it. And then a bunch of friends more around the edges.

How about you and @mattbk decide together what you want to each take, and then the rest of the folks can fit in around that? As I say, I'm happy to park at 1¢ for the foreseeable future.

What are other folks seeing? @JessaWitzel @dmk246 @EdOverflow @sseerrggii @lurtz et al. @gratipay?

**Comment by whit537** [comment]
P.S. Some notes on an experiment with the rankings-based approach:

http://magazine.ouishare.net/2015/11/decentralizing-part-of-ouishare-with-blockchain-experiment-1/
http://magazine.ouishare.net/2016/01/how-to-slowly-lose-control-ouishare-decentralization-experiment-chapter-2/

**Comment by JessaWitzel** [comment]
I absolutely think your take is impacting everyone else's but "not needing money" is not really a compelling reason to drop yourself down to 1c. My bills are paid but I still thing that I should be compensated for work that I do if it's helping. I am getting more than I am giving right now and am going to keep my take at .10. If I finish a project I might go to $1 that week to "reward" myself and motivate myself to finish more projects/make more commits.

**Comment by JessaWitzel** [comment]
Also very interesting conversation and thoughtfulness about twyw. If we have this issue other teams will too! :)

**Comment by nobodxbodon** [comment]
> What do you want to take, @nobodxbodon? Personally I see you and @mattbk doing the most work besides me—probably even as much as me. I see another tier of folks doing good work but a bit less of it. And then a bunch of friends more around the edges.

Although I'm not going to take due to my current status, it'll be rewarding for me to see my peers' view of my contribution, just like your view above, which I don't see in current twyw practice.

Below I'm trying to address from my view the issues listed in the experiement post above:

> Practical questions

> 1. Will we be spending all our time evaluating people’s work? One of the core features of the Backfeed application is that each time a task is completed, others can evaluate it by giving out tokens. For the system to properly work, a critical mass of reputation-holding members must evaluate all contributions. But how much additional work will this new process generate? How can the tool be integrated into an existing workflow, to avoid creating more overhead for the team? This has been one of the largest concerns of team members. Making the step to a new system of organization takes extra effort, especially in the beginning. This is an investment that will hopefully pay off in efficiency later on, but there is no way to guarantee that it will.

In our current twyw practice, seems people are already ranking everyone's work, thus I don't see much additional workload other than quantifying that ranking.

> 2. How do you integrate on and offline coordination? There is the tendency to view tech as a replacement for offline social interaction, seeing it as parallel but separate world. We often forget that just because we are using an online tool for coordination, this does not mean we cannot coordinate offline. The missing link is how to integrate offline actions with digital tools in an organic, fluid way, an aspect that has been overlooked in the discourse around decentralized organization so far.

Similar to the answer above, currently individual's evaluation is already based on overall work, very subjectively but I don't think we need to measure everything to get a relatively fair evaluation.

> Ideological questions

> 1. Is it a good thing to evaluate everything we do? If we do in fact start continuously evaluating everyone’s work as described above, do we risk putting too much performance pressure on ourselves? How free are we really if our every move is documented, assessed and quantified? Will we start seeing people more as numbers than humans?

Ditto.

> 2. How should reputation be distributed? To start using the Backfeed system, you need to define two things; 1) who are the people who will hold the initial reputation (referred to as founding members), and 2) how will that reputation be distributed among them. Over time, the founding members’ reputation will be transferred to other contributors, based on how the founding members evaluate others’ contributions. Power is in the hands of few at the beginning and is distributed over time, a process of reputation transfer Backfeed also described as “losing control”. This raised a huge amount of debate within the team: what entitles someone to receive reputation? What warrants one person to receive more reputation than another? How far back in your organization’s history do you go to look for evidence of contribution?

Maybe better to evaluate contribution over "reputation". And in short time period (week), it should be easier to evaluate, than taking into account the whole organization's history. Besides, a weekly gathering to review the work done will be rewarding in itself.

> 3. What kind of behavior do we want to incentivize? Could this system change our behavior in ways we do not want? How do the constraints of the system limit our potential behavior? What motivates different people in our team? Will people really respond to the same incentives in the same ways?

I don't see any clear problem for now but maybe others have?

> 4. To what extent can code impose a certain world view? Should we be implementing a system that is so complex that we do not fully understand it, or the values embedded within it?

Same applies to current TWYW.

> 5. What is “value” for us? What criteria do we take into account to evaluate tasks? How do we establish the relative value of tokens (which is what people receive when they are evaluated) in a consistent way? If this is not something we collectively pre-establish, how long will it take for a common sense of value to emerge among the group

As mentioned above, two values I see 1) peer review of your contribution 2) regular review of progress and understand others' accomplishments

**Comment by nobodxbodon** [comment]
About "[3 lessons from Gratipay's take-what-you-want compensation experiment](https://opensource.com/open-organization/16/7/compensating-employees-letting-them-take-what-they-want)", I don't see the outcome of the short-try of ranking-based approach. Maybe I missed?

BTW my suggestion was not to replace TWYW, but making it easier to make decision about what to take, by having others' view.

Below is an imaginary simplified procedure:

It's a Friday, and it's time to do review of this week's work. There are 3 active collaborators A, B, C, who either working on a PR, or reviewed, or discussed, or carried out managing/marketing or any work that the collaborators are willing to share. Everyone writes a summary and they are aggregated to a weekly report.

Each of the 3 collaborators check the report, and give their own perspective in distribution, with remaining weekly budget $50 in mind:

| peer | A's share | B's share | C's share |
| ------------- | ------------- | ------------- | ------------- | 
| A | $10 | $15 | $20 |
| B | $10 | $10 | $10 |
| C | $0 | $0 | $50 |
| Average | $6.67 | $8.33 | $26.67 |

Based on the average outcome and the peer's view, A decides to take $10, B takes $0, and C takes $5

**Comment by mattbk** [comment]
Without too much navel-gazing, I'm just going to divide the amount available ($50) by the number of people on the team (9) and take that. Now at $5.55.

In the spirit of TWYW, I'll try to respect everyone's choice of take in the future. Growth mindset and all that--there will be enough for everyone!

**Comment by whit537** [comment]
Love it, @JessaWitzel @mattbk! 💃  Thanks for explaining your approach and how you see things. 👍

> As mentioned above, two values I see 1) peer review of your contribution 2) regular review of progress and understand others' accomplishments

Don't we already have regular peer review and understanding of others' accomplishments? At our scale, GitHub and Slack seem fine to me. Perhaps we can bring back the [Monday Morning Missive](https://tinyletter.com/gittip) when we have enough people involved again.

I am highly uninterested in being asked to assign dollar values to all of yinz' work, as that would drastically change the psychology of the situation. If it's hard enough to choose where to set your own take now, imagine how much harder it would be to feel self-determined if the rest of us told you explicitly how much _we_ thought you should be taking! As demonstrated on this thread, each of us has our own unique algorithm for deciding what is the right amount for us personally. With enough flexibility in our ranges of "fair" regarding each other, and good communication, we can find a balance point. If we start _fixing_ what is "fair" for others, then we loose the necessary flexibility and get into bad mojo.

> I'm not going to take due to my current status

Sounds like a +1 for https://github.com/gratipay/gratipay.com/issues/4299. ;-)

> Actually, I think he's the last anyway if TWYW is still operating under "kids eat first," where the newest members of the team project get first crack at taking.

P.S. Kids eat first was twyw 1.0. Under 2.0 it's smallest to largest—so now that I'm at 1¢ I'm actually first to receive! The order on the [page](https://gratipay.com/Gratipay/distributing/) is the order we receive in.

**Comment by whit537** [comment]
Are we ready to close, here? Anyone else want to weigh in?

**Comment by mattbk** [comment]
> P.S. Kids eat first was twyw 1.0. Under 2.0 it's smallest to largest—so now that I'm at 1¢ I'm actually first to receive! The order on the page is the order we receive in.

Awesome, thanks!

**Comment by nobodxbodon** [comment]
> If it's hard enough to choose where to set your own take now, imagine how much harder it would be to feel self-determined if the rest of us told you explicitly how much we thought you should be taking! 

I don't want to drag this into a longer discussion either, but the hardness to set own take now IMO is largely due to lack of understanding of others' view of your accomplishments. Maybe I'm totally wrong about this, but I think everyone is already trying to take others view into account when making the decision, but just guessing without any direct confirmation. After all, the extent of self-determination is largely restricted by social factors, implicitly or explicitly, regardless of which distribution model.

> As demonstrated on this thread, each of us has our own unique algorithm for deciding what is the right amount for us personally. 

Current practice seems to put all the burden of decision on individual instead of sharing the burden within the community. The fact that member has to resort to egalitarianism, "divide the amount available ($50) by the number of people on the team (9)" (BTW the page seems outdated, according to my feeling of contributing recently) regardless of his accomplishments just doesn't look right.

> P.S. Kids eat first was twyw 1.0. Under 2.0 it's smallest to largest—so now that I'm at 1¢ I'm actually first to receive! The order on the page is the order we receive in.

Is there an order of 'saying' what he/she's going to take? Would it happen that everyone waits for you to announace your take first?

**Last but not the least, not just because I won't take in the short term, I really don't mind closing this now, and focusing on the epic.**

**Comment by mattbk** [comment]
@nobodxbodon, I think we have to accept some sort of uncertainty with this grand TWYW experiment. I know I'm the one who brought this up, and it was because I was uncertain of what I _should_ be taking. After the discussion, though, I find that I'm more accepting of this uncertainty than I am of directly scoring my teammates.

It's easier to be uncertain and accepting of things now, because the stake is so low at the moment. We can think of it as practice for when more money enters the system.

Finally, with the new smallest-to-largest order, it puts a brake on people taking too much on the top end. Sure, I could take $50 now and end up with nearly that, but you could come along and take $49, and I wouldn't get any.

**Comment by whit537** [comment]
> I was uncertain of what I _should_ be taking. 

You should be taking whatever balances your own internal guilt and resentment ~= sense of fairness. Every other possible variable is a proxy of this one. The goal of twyw is to optimize fairness across the team, and it _depends_ on uncertainty, as mentioned above in my comment about the psychology of the situation.

> I find that I'm more accepting of this uncertainty than I am of directly scoring my teammates.

👍 💯 

> We can think of it as practice for when more money enters the system.

The way Morning Star (400-person company with self-set salaries for 30 years) handles this is with a review board that gets in touch with you if your annual salary increase is out of line. There _is_ some precedent and I think we'll be able to figure this out as we scale.

> I really don't mind closing this now, and focusing on the epic.

👍

**Comment by whit537** [comment]
> Current practice seems to put all the burden of decision on individual instead of sharing the burden within the community.

I would also say that if/when you are ready to start taking money, @nobodxbodon, feel free to open a ticket here to discuss where to set your take. The decision is yours, but you can of course ask the rest of us for advice. :)
